Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -469,7 +469,7 @@ void PreferencesDlg::slotTabChange(int index)
     {
     case GENERAL_TAB : //general
     {
-        ui.nicknameEdit->setText(ttSettings->value(SETTINGS_GENERAL_NICKNAME, QCoreApplication::translate("MainWindow", SETTINGS_GENERAL_NICKNAME_DEFAULT)).toString());

```

Comment:
Keep SETTINGS_GENERAL_NICKNAME_DEFAULT but set SETTINGS_GENERAL_NICKNAME_DEFAULT to an empty string